src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -0,0 +1,71 @@
+package io.iohk.ethereum.sync
+
+import io.iohk.ethereum.FlatSpecBase
+import io.iohk.ethereum.sync.util.RegularSyncItSpecUtils.FakePeer
+import io.iohk.ethereum.sync.util.SyncCommonItSpec._
+import monix.execution.Scheduler
+import org.scalatest.BeforeAndAfter
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.duration._
+
+class RegularSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfter {
```

Comment:
can we change  `BeforeAndAfter` to `BeforeAndAfterAll` and then add :
```
  override def afterAll(): Unit = {
    testScheduler.shutdown()
    testScheduler.awaitTermination(60.second)
  }
```

? ( we should probably do this also in `FastSyncItSpec`)

Reply:
Done